src/helpers/_media-queries.scss

```diff
@@ -1,4 +1,12 @@
-// Import sass-mq as a helper for media queries
+////
+/// @group helpers
+////
+
+
+
+// =========================================================
+// Wrangle sass-mq config...
+// =========================================================
 
 // Pass our breakpoints and static breakpoint definitions through to sass-mq.
 $mq-breakpoints: if(variable-exists(govuk-breakpoints), $govuk-breakpoints, ());
```

Comment:
Because we do this configuration 'at import time', any changes to `$govuk-is-ie8`, `$govuk-breakpoints`, `$govuk-ie8-breakpoint` made after the import has been done will not have any effect on `govuk-media-query`. We _could_ move this wrangling into the `govuk-media-query` mixin, but it complicates the way we pass arguments somewhat, and relies on us being able to pass that config to `mq` which is an undocumented feature, and I can't think of any valid use cases where you'd want to change that config mid-way through your codebase.

Thoughts?